specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -981,7 +981,7 @@
         "adminUsername": {
           "type": "string",
           "description": "The administrator username to use for Linux VMs.",
-          "pattern": "^[a-z][a-z0-9_-]*$"
+          "pattern": "^[A-Za-z][-A-Za-z0-9_]*$"
```

Comment:
Your current input files are:
```
input-file:
- Microsoft.ContainerService/stable/2017-07-01/containerService.json
- Microsoft.ContainerService/stable/2018-03-31/managedClusters.json
- Microsoft.ContainerService/stable/2017-09-30/location.json
```
and this PR creates this error:
```
INFO:swaggertosdk.autorest_tools:Process() cancelled due to exception : '$.definitions.ContainerServiceLinuxProfile.properties.adminUsername.pattern' has incompatible values (---
INFO:swaggertosdk.autorest_tools:'^[a-z][a-z0-9_-]*$'
INFO:swaggertosdk.autorest_tools:, ---
INFO:swaggertosdk.autorest_tools:'^[A-Za-z][-A-Za-z0-9_]*$'
INFO:swaggertosdk.autorest_tools:).
```

which means to me that containerService or location files defines this same object, and needs to be updated as well, or the model renamed to avoid conflicts.